test/e2e/framework/node/resource.go

```diff
@@ -47,6 +46,21 @@ const (
 	sshPort = "22"
 )
 
+var (
+	// UnreachableTaintTemplate is the taint for when a node becomes unreachable.
+	UnreachableTaintTemplate = &v1.Taint{
```

Comment:
This is used within this file, so we can define this as a local with `unreachableTaintTemplate`.